topology/probes/libvirt/libvirt.go

```diff
@@ -112,7 +112,8 @@ func (probe *Probe) getDomainInterfaces(
 	for _, itf := range d.Interfaces {
 		if constraint == "" || constraint == itf.Alias.Name {
 			itf.Host = domainNode
-			interfaces = append(interfaces, &itf)
+			itf_obj := itf
```

Comment:
The golinter will raise an error because of "_" use. Could you please use itfObj instead, thanks

Reply:
Of course. This is a normative question, you are right.